### ISSUE TITLE
[Helix] LoggedTests failing with Null Ref on Helix Windows queue

### ISSUE THREAD

**Issue by HaoK** [aspnet/AspNetCore#7847]
Lots of logged tests are failing with exceptions like this only on Windows/Helix queue

Disabled tests include Kestrel/test/BindTests/* and FunctionalTests/*
StaticFiles.FunctionalTests
WebSockets/UnitTests

```
Failed   RegisterAddresses_IPv6Port80_Success(addressInput: "http://localhost", testUrls: ["http://127.0.0.1/", "http://[::1]/"])
Error Message:
 System.NullReferenceException : Object reference not set to an instance of an object.
Stack Trace:
   at Microsoft.Extensions.Logging.Testing.LoggedTest.Initialize(MethodInfo methodInfo, Object[] testMethodArguments, ITestOutputHelper testOutputHelper)
   at Microsoft.AspNetCore.Testing.TestApplicationErrorLoggerLoggedTest.Initialize(MethodInfo methodInfo, Object[] testMethodArguments, ITestOutputHelper testOutputHelper) in /_/src/Servers/Kestrel/shared/test/TestApplicationErrorLoggerLoggedTest.cs:line 16
   at Microsoft.Extensions.Logging.Testing.LoggedTestInvoker.CreateTestClass()
Failed   RegisterAddresses_IPv6Port80_Success(addressInput: "http://[::1]", testUrls: ["http://[::1]/"])
Error Message:
 System.NullReferenceException : Object reference not set to an instance of an object.
Stack Trace:
   at Microsoft.Extensions.Logging.Testing.LoggedTest.Initialize(MethodInfo methodInfo, Object[] testMethodArguments, ITestOutputHelper testOutputHelper)
   at Microsoft.AspNetCore.Testing.TestApplicationErrorLoggerLoggedTest.Initialize(MethodInfo methodInfo, Object[] testMethodArguments, ITestOutputHelper testOutputHelper) in /_/src/Servers/Kestrel/shared/test/TestApplicationErrorLoggerLoggedTest.cs:line 16
   at Microsoft.Extensions.Logging.Testing.LoggedTestInvoker.CreateTestClass()
Failed   RegisterAddresses_IPv6Port80_Success(addressInput: "http://*", testUrls: ["http://[::1]/"])
Error Message:
 System.NullReferenceException : Object reference not set to an instance of an object.
Stack Trace:
   at Microsoft.Extensions.Logging.Testing.LoggedTest.Initialize(MethodInfo methodInfo, Object[] testMethodArguments, ITestOutputHelper testOutputHelper)
   at Microsoft.AspNetCore.Testing.TestApplicationErrorLoggerLoggedTest.Initialize(MethodInfo methodInfo, Object[] testMethodArguments, ITestOutputHelper testOutputHelper) in /_/src/Servers/Kestrel/shared/test/TestApplicationErrorLoggerLoggedTest.cs:line 16
   at Microsoft.Extensions.Logging.Testing.LoggedTestInvoker.CreateTestClass()
Failed   ListenAnyIP_IPv6_Success
Error Message:

```

**Comment by pakrym** [comment]
cc @JunTaoLuo

**Comment by pakrym** [comment]
I think I know what caused this issue. We compile log directory into test assemblies so when they are published to helix the path is wrong. Not sure how it worked before.

**Comment by HaoK** [comment]
Its only broken on windows too, and working fine on the other queues

**Comment by JunTaoLuo** [comment]
Hmm how are the agents configured? We used to have explicit [environment variables](https://github.com/aspnet/Extensions/blob/master/src/Logging/Logging.Testing/src/build/Microsoft.Extensions.Logging.Testing.props#L3-L6) configured in teamcity.

**Comment by JunTaoLuo** [comment]
This has come up a few times now. We really need a better error message.

**Comment by HaoK** [comment]
Helix so no configuration really, these tests were passing previously (on an older dotnet runtime), but we can try adding these variables to see if the might help

**Comment by pakrym** [comment]
I've disabled file logging in my PR (there is currently no way to get them of machines anyway). I'll ping this thread if it helps.

**Comment by pakrym** [comment]
Disabling logs helped.

**Comment by HaoK** [comment]
Cool I'll add that to my PR then and reenable the tests I skipped

**Comment by HaoK** [comment]
Was fixed by 78bc2a10b9ea617269527f4091903471a299e197
